src/shared/processors/low/image.js

```diff
@@ -52,7 +52,11 @@ export default {
     let src = null;
 
     // Get src attribute from different cases.
-    if (props.src && typeof props.src === 'string') {
+    if (props['data-src'] && typeof props['data-src'] === 'string') {
+      if (props.src.startsWith('/'))
+        src = `${settings.generalSite.url}${props.src}`;
```

Comment:
Shouldn't this be:
```js
if (props['data-src'].startsWith('/'))
        src = `${settings.generalSite.url}${props['data-src']}`;
```
?

If that's the case, add a test as that part of the code is not covered.